packages/components/src/button/src/ButtonGroup.tsx

```diff
@@ -39,6 +39,7 @@ export function InnerButtonGroup(props: InnerButtonGroupProps) {
         children,
         disabled,
         fluid,
+        wrap = true,
```

Comment:
I might be missing some context so please forgive me if I'm not understanding it correctly!
We had this discussion a long time ago also with Alex, but I'm not a big fan of booleans defaulting to `true` in props.

That means that if I want the `InnerButtonGroup` to be no wrap, I don't to specifically say `wrap={false}`. In other words: `<InnerButtonGroup wrap={false}>` works but `<InnerButtonGroup wrap>` is useless.

In that case would it be ok to with `noWrap = false` here, and on line 71 something along the lines of `wrap: !noWrap`.
Or maybe `wrap` (with no default set on line 42) and, on line 71, `wrap: noWrap ? undefined : true`.

Opening this up for discussion, ultimately the solution you have now works just fine so not bothered if we go with it!

Reply:
I am open to any of these solutions, what do you think @alexasselin008

Reply:
In this case, the props with the name wrap are in multiple components. Therefore, i wouldnt not change the name, even if i agree with what you said about true default values.